content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -48,82 +48,97 @@ For a full list of supported libraries and processor architectures, see [Compati
 
 ## Installation and getting started
 
-### Automatic instrumentation
-
 <div class="alert alert-warning">
-  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li></ul>
 </div>
 
-#### Installing the tracer
+### Installation overview
+
+1. Install the tracer
+2. Enable the tracer for your service
+3. Configure the Datadog Agent for APM
+4. View your live data
+
+### Install the tracer
+
+The Datadog .NET Tracer can be installed machine-wide to allow services across an entire machine to be instrumented, or it can be installed on a per-application-basis to allow developers to manage the instrumentation via the application’s dependencies. To see machine-wide installation options, view the Windows or Linux tab. To see per-application installation options, see the NuGet tab.
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
+To install the .NET Tracer:
+
 1. Download the [.NET Tracer MSI installer][1]. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
 
 2. Run the .NET Tracer MSI installer with administrator privileges.
 
-3. Enable instrumentation in your service by setting the required environment variables. See the section *Instrumenting your service*, below.
 
-4. Create application load.
+[1]: https://github.com/DataDog/dd-trace-dotnet/releases
+{{% /tab %}}
+
+{{% tab "Linux" %}}
+
+To install the .NET Tracer:
 
-5. Visit [APM Live Traces][2].
+1. Download the latest [.NET Tracer package][1] that supports your operating system and architecture.
+
+2. Once downloaded, install the package with one of the commands below:
+
+```bash
```

Comment:
This section was adapted from the in-app docs and put into one code block for brevity, similar to the PHP's [extension installation section](https://docs.datadoghq.com/tracing/setup_overview/setup/php?tab=containers#install-the-extension). This can be reverted to the separate code blocks that we had before if that's preferred

Reply:
I wonder if we should include the "Run the `/opt/datadog/createLogPath.sh` script" section in these commands - I've seen several cases where people haven't, and I think this would improve onboarding?

Reply:
This is a good idea. I updated it with a revision, let me know what you think

Reply:
I like it, but [I wonder if a one-liner would be better](https://github.com/DataDog/documentation/pull/11627/files#r705240623) (It's easier & better to copy into to Docker files for example).

As an aside, we need to update our in-app docs for linux host installs, as they don't mention this step at all!